src/main/java/core/basesyntax/service/ReportCreator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.ReadFromCsvFileImpl;
+
+public interface ReportCreator {
+    String createReport(ReadFromCsvFileImpl file);
+}
```

Comment:
```suggestion
public interface FruitStorageService {
    void saveFruitToStorage(List<String> lines); // lines that were read by the reader
    
    String createReport();
}
```
let's refactor this service